### PR TITLE
Work branch to remove gorpc support.

### PR DESCRIPTION
**Pull request by alainjobart**
Will keep adding to it.

---
This change is [Reviewable](https://reviewable.io/reviews/youtube/vitess/1574)

### CODE REVIEW

**Comment by alainjobart** [comment]
@enisoc @aaijazi @dumbunny PTAL this is ready for review.

(I think I'd like to import this today before internal cut).

File: go/vt/vtgate/txbuffer/tx_buffer_test.go [hunk 1]
```diff
@@ -149,23 +152,30 @@ func TestParallelFakeBuffer(t *testing.T) {
...
+			block:   true,
+			done:    make(chan struct{}),
+			blocked: make(chan struct{}),
 		}
 		timeSleep = createFakeSleep(controller)
 		// Only the first maxBufferSize calls to FakeBuffer should actually call fakeSleep
 		wantFakeSleepCalled := (i <= *maxBufferSize)
 
 		wg.Add(1)
+		finished := make(chan struct{})
```
Comment (aaijazi) [inline go/vt/vtgate/txbuffer/tx_buffer_test.go]: Cool, thanks for doing this!

LGTM

**Comment by enisoc** [comment]
:lgtm:

---

Reviewed 3 of 5 files at r1, 12 of 12 files at r2, 10 of 10 files at r3, 10 of 11 files at r4, 4 of 4 files at r5, 15 of 15 files at r6, 7 of 7 files at r7, 3 of 3 files at r8, 28 of 28 files at r9, 37 of 37 files at r10, 1 of 1 files at r11.
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/youtube/vitess/1574#-:-KD0DzUzVCcet9z7Gras:1417088721)*

**Comment by enisoc** [comment]
Pushed new bootstrap and lite images, tested k8s guide.

**Comment by enisoc** [comment]
Published site.

**Pull request merged**